.github/workflows/composite/setup-docker-ci/action.yml

```diff
@@ -0,0 +1,54 @@
+name: 'Setup Docker CI Environment'
+description: 'Creates the LunaSec CI build environment'
+
+inputs:
+  merge_master:
+    description: 'If true, this merges the master branch during setup.'
+    required: true
+  skip_docker:
+    description: 'If true, skips the Docker install step'
+    required: true
+  event_name:
+    description: Should just be `github.event_name`
+    required: true
+
+runs:
+  using: "composite"
+#  env:
+#    # avoid warnings like "tput: No value for $TERM and no -T specified"
+#    TERM: xterm
+  steps:
+    - name: Set up Docker BuildKit
+      id: buildx
+      uses: docker/setup-buildx-action@v1
+      with:
+        install: true # sets buildx as the default for docker, which should apply to docker-compose commands
+        driver: docker
+#      if: ${{ inputs.skip_docker != 'false' }}
+
+    - uses: actions/setup-node@v2
+      with:
+        node-version: '14'
+        cache: yarn
+#      if: ${{ inputs.skip_docker != 'false' }}
+
+    # specifically tag this bootstrap container to prevent rebuilds
+    - name: Build Lerna Bootstrap container
+      shell: bash
+      env:
+        # avoid warnings like "tput: No value for $TERM and no -T specified"
+        TERM: xterm
+        DOCKER_BUILDKIT: 1
+        COMPOSE_DOCKER_CLI_BUILD: 1
```

Comment:
Can we not put these at the top anymore because of composite mode?

Reply:
I'll test this to verify.